modules/core/src/v2/wallet.ts

```diff
@@ -1168,8 +1171,12 @@ Wallet.prototype.prebuildAndSignTransaction = function(params, callback) {
     // call prebuildTransaction and keychains-get in parallel
     // the prebuild can be overridden by providing an explicit tx
     const txPrebuildQuery = params.prebuildTx ? Promise.resolve(params.prebuildTx) : this.prebuildTransaction(params);
+
     const userKeychainQuery = this.baseCoin.keychains().get({ id: this._wallet.keys[0], reqId: params.reqId });
-    const [txPrebuild, userKeychain] = yield Promise.all([txPrebuildQuery, userKeychainQuery]);
+    const backupPublicKeys = this.baseCoin.keychains().get({ id: this._wallet.keys[1], reqId: params.reqId });
+    const bitgoPublicKey = this.baseCoin.keychains().get({ id: this._wallet.keys[2], reqId: params.reqId });
```

Comment:
So we have to get the bitgo key for all coins now? :(

Reply:
all pubs for algo

Reply:
ok changed all this around